### ISSUE TITLE
Evaluation about AutoML Methods

### ISSUE THREAD

**Issue by pingguokiller** [MedMNIST/MedMNIST#8]
Thanks very much for your nice work!

I have read your source code and paper. I found both your code and AutoKeras chose the best model based on the highest AUC score on the validation set. 

However, how Google AutoML Vision2 and auto-sklearn evaluate is introduced. If they use the best test auc/acc during the searching, is it an unfair comparison?

**Comment by duducheng** [comment]
Hi,

Thanks for the interests in our study!

In all experiments, we only make Train and Val set visible to the algorithms during training, including Google AutoML Vision and auto-sklearn for sure. The algorithms never know anything about test set until the evaluation phase.
